docs/usage/configuration-options.md

```diff
@@ -2036,10 +2036,11 @@ Consider this example:
 
 With the above config, every PR raised by Renovate will have the label `dependencies` while PRs containing `eslint`-related packages will instead have the label `linting`.
 
-Renovate only adds labels when it creates the PR, which means:
+Behaviour details:
 
+- Renovate only adds labels when it creates the PR
 - If you remove labels which Renovate added, it won't re-apply them
-- If you change your config, the new/changed labels are not applied to any open PRs
+- If you change your config, the new/changed labels are applied to any open PRs
```

Comment:
When you say "change your config", do you mean:

- Change label name in Git hosting  platform
- Change label config option in Renovate config file
- Something else

Reply:
Change label config option in Renovate config file

Reply:
```suggestion
- If you change the `label` config option, the new/changed labels are applied to any open PRs
```

Reply:
I think the _before_ is better, because we can also change labels using `addLabels` and within `packageRules`.